packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/create-local-file-node.js

```diff
@@ -90,16 +96,17 @@ export const errorPanicker = ({
 
   if (errorString.includes(`Response code 4`)) {
     reporter.log(``)
+
     reporter.info(
       formatLogMessage(
         `Unrecoverable error ${sharedError}\n\nFailing the build to prevent deploying a broken site.${
-          errorString.includes(`Response code 404`)
-            ? `\n\nIf you don't want 404's to fail your production builds, you can set the following option:
+          errorCode
+            ? `\n\nIf you don't want ${errorCode}'s to fail your production builds, you can set the following option:
 
 {
   options: {
     production: {
-      allow404Images: true
+      allow${errorCode}Images: true
```

Comment:
This could be potentially misleading for 40x codes that aren't 404 or 401. https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors

I wonder if we should instead add a new plugin option that allows the user to bypass any error code by adding it to an array.

```
 options: {
    production: {
      allowImageErrorCodes: [404, 401]
```

Not sure though 🤔 

If these are the only two codes folks will ever care about I think the way you did it is good but we'll want to make the check here specific to 404's and 401's

Reply:
Also I'm just realizing I should've named it `allow404Files` 🤦 😛

Reply:
Yea I think making it more flexible like that in the long run is a good idea.  But can we move with this as is since it is opt-in?

Reply:
Usually I want to try to avoid adding plugin options since it's much harder to remove them than to add them but I think in this case no one has asked for additional codes so the extra flexibility maybe isn't needed. Lets keep it like this 👍 

I just looked at the code again and I guess your condition only applies to 404/401. I didn't realize the errorCode can only be one of those so we should be good!